### DISCUSSION TITLE
Thoughts

### DISCUSSION THREAD

**Discussion by plutack1** [robertodoering/harpy#549] [General]
Can the timeline be sorted by time? Also provision for addition of multiple profiles/users.
Ps: great work here it's super impressive

**Comment by robertodoering** [comment]
Thank you!
Support for multiple accounts is planned, here is the issue: #249

The timelines are sorted chronologically by default. Exceptions are when a tweet in the timeline is replied to by another tweet in the same timeline. In that case the reply will push the older tweet up in the timeline, so that the newest reply will always determine the position of the original tweet in the timeline.

**Reply by plutack1** [reply]
Off topic: it will be dope if we can get a premium reddit client with the same vibes.  I understand the work of developer is not exactly easy. Just saying though

**Reply by plutack1** [reply]
I also figured out to reproduce exactly what I wanted after reading your reply by excluding retweets.

**Reply by robertodoering** [reply]
Right, retweets will be displayed in the timeline at the time of the retweet, so the date on the retweeted tweet is a bit misleading.

A reddit client sounds interesting and you are actually not the first one to suggest that. However there are already a number or pretty good 3rd party reddit clients already so it might not make a lot of sense for me to work on one as well.

**Reply by plutack1** [reply]
That's alright I guess. I also noticed there isn't the notification alert for people you follow you want to be notified about their tweets option in the app

**Reply by plutack1** [reply]
I should also mention. I noticed a user pinned tweet isn't isolated.
Also is there a telegram group or something for suggestions and observations?

**Reply by robertodoering** [reply]
There is no telegram group, sending your feedback through GitHub is preferred

**Reply by plutack1** [reply]
Ok not a problem.
I also notice I can't block a user

**Comment by plutack1** [comment]
And I hope it's cool mentioning my suggestions and observations on this thread I'm not too familiar with GitHub.

**Reply by robertodoering** [reply]
Yeah definitely, you can also take a look at the project https://github.com/robertodoering/harpy/projects/1
there you'll find upcoming features and known issues

**Reply by plutack1** [reply]
Alright
